cmd/mt-whisper-importer-reader/main.go

```diff
@@ -109,11 +109,12 @@ func main() {
 	fileChan := make(chan string)
 
 	wg := &sync.WaitGroup{}
+	wg.Add(*threads)
```

Comment:
why does the commit message claim to not use a waitgroup, clearly we do. we just use it in a better way.